examples/spacy/train.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+
+import random
+
+import spacy
+from spacy.util import minibatch, compounding
+
+import mlflow.spacy
+
+# training data
+TRAIN_DATA = [
+    ("Who is Shaka Khan?", {"entities": [(7, 17, "PERSON")]}),
+    ("I like London and Berlin.", {"entities": [(7, 13, "LOC"), (18, 24, "LOC")]}),
+]
+
+if __name__ == "__main__":
+    # Adaptation of spacy example: https://github.com/explosion/spaCy/blob/master/examples/training/train_ner.py
+
+    # create blank model and add ner to the pipeline
+    nlp = spacy.blank("en")
+    ner = nlp.create_pipe("ner")
+    nlp.add_pipe(ner, last=True)
+
+    # add labels
+    for _, annotations in TRAIN_DATA:
+        for ent in annotations.get("entities"):
+            ner.add_label(ent[2])
+
+    metrics = {
+        'n_iter':100,
+        'drop': 0.5
+    }
+    mlflow.log_params(metrics)
+
+    nlp.begin_training()
+    for itn in range(metrics['n_iter']):
+        random.shuffle(TRAIN_DATA)
+        losses = {}
+        # batch up the examples using spaCy's minibatch
+        batches = minibatch(TRAIN_DATA, size=compounding(4.0, 32.0, 1.001))
+        for batch in batches:
+            texts, annotations = zip(*batch)
+            nlp.update(
+                texts,  # batch of texts
+                annotations,  # batch of annotations
+                drop=metrics['drop'],  # dropout - make it harder to memorise data
+                losses=losses,
+            )
+        print("Losses", losses)
+        mlflow.log_metrics(losses)
+
+    # load it again (just as an example) and log metrics
+    mlflow.spacy.log_model(spacy_model=nlp, artifact_path='model')
+    model_uri = "runs:/{run_id}/{artifact_path}".format(
+        run_id=mlflow.active_run().info.run_id,
+        artifact_path='model')
```

Comment:
Is the comment here correct?

Reply:
It was intended for some of the lines below (starting from 52), but I see how that could be a little misleading. Changed them a little in the latest commit.
Thanks